### ISSUE TITLE
sqlrpgle missing from package.json

### ISSUE THREAD

**Issue by Dustii77** [barrettotte/vscode-ibmi-languages#1]
Simple little thing. 

![image](https://user-images.githubusercontent.com/54592861/77862498-5112fa80-71e1-11ea-8731-cb9784e231ac.png)

**Comment by barrettotte** [comment]
Ah my fault I should have waited to release the extension until I finished basic RPGLE highlighting. The new version of this extension (0.4.0) that I published today should fix this issue.

Specifically this commit added the extension recognition 
https://github.com/barrettotte/vscode-ibmi-languages/commit/232a4404b243e37c9c1741a1d65a6dbf1f2d48c6#diff-b9cfc7f2cdf78a7f4b91a753d10865a2

In the future I will do active development in a separate branch, so that master reflects what is currently published in case that was confusing.

I'm surprised anyone found this extension, thanks for trying it out! It gives me more motivation to improve it
